testsuite/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+#![no_std]
+#![cfg_attr(test, no_main)]
```

Comment:
I've had to add `no_main` because lib.rs will be compiled as an executable when calling `cargo test`.